docs/minio-admin-complete-guide.md

```diff
@@ -289,18 +289,33 @@ Skip SSL certificate verification.
 
 <a name="update"></a>
 ### Command `update` - updates all MinIO servers
-`update` command provides a way to update all MinIO servers in a cluster.
-
-> NOTE:
-> - An alias pointing to a distributed setup this command will automatically update all MinIO servers in the cluster.
-> - `update` is an disruptive operations for your MinIO service, any on-going API operations will be forcibly canceled. So, it should be used only when you are planning MinIO upgrades for your deployment.
+`update` command provides a way to update all MinIO servers in a cluster. You can also pass a custom download server URL to `update` command. This is useful in cases where MinIO is running in an environment that doesn't have Internet access.
 
 *Example: Update all MinIO servers.*
 ```
 mc admin update play
 Server `play` updated successfully from RELEASE.2019-08-14T20-49-49Z to RELEASE.2019-08-21T19-59-10Z
 ```
 
+### Steps to update MinIO using a custom download server
+You can use any webserver as the download server for `mc` update. For `update` using custom download server for MinIO binary, you need to have two files:
+
+- Release information file in a format like this [http://dl.minio.io/server/minio/release/linux-amd64/minio.sha256sum]
+- Actual MinIO binary to be downloaded. This file should be named as `minio` for Linux environments and `minio.exe` for Windows environments.
+
+For example, if your custom download server is running at `localhost:8000`, and the release information file is available as `localhost:8000/release_info`, the MinIO binary should be available at `localhost:8000/minio`.
+
+*Example: Update all MinIO servers using custom download server.*
```

Comment:
https://github.com/minio/minio/pull/9111#issuecomment-596988656

Some explanation on why this is not correct and we should mention how to do a centralized repo for all platforms.

Reply:
Updated the PR, PTAL